bindings/python/libmata/nfa/strings.pxd

```diff
@@ -0,0 +1,38 @@
+from libcpp cimport bool
+from libcpp.set cimport set as cset
+from libcpp.string cimport string
+from libc.stdint cimport uintptr_t
+from libcpp.unordered_map cimport unordered_map as umap
+from libcpp.memory cimport shared_ptr
+from libcpp.vector cimport vector
+
+from libmata.nfa.nfa cimport CNfa, CTrans
+
+cdef extern from "mata/nfa.hh" namespace "Mata::Nfa":
+    ctypedef vector[CTrans] TransitionSequence
+    ctypedef vector[CNfa] AutSequence
+    ctypedef vector[CNfa*] AutPtrSequence
+    ctypedef vector[const CNfa*] ConstAutPtrSequence
+    ctypedef umap[string, string] StringMap
+
+cdef extern from "mata/alphabet.hh" namespace "Mata":
+    ctypedef uintptr_t Symbol
```

Comment:
Why do you have to declare the type for symbols again here if you already have this type in `alphabets.pxd`?

Reply:
Due to the failing-compilation despair. The Cython is quite a wobbly ecosystem always on the verge of mental breakdown. The `from libmata.alphabets cimport Symbol` should do the work. I will change the similar lines in other modules as well.